feat(curriculum): add debugging registration form workshop (#66689)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/workshop-debugging-registration-form/69ca8ef06031d0065893012b.md

---
id: 69ca8ef06031d0065893012b
title: Step 2
challengeType: 0
dashedName: step-2
---

# --description--

> As the editor do not support certain methods, you will be using the browser console to view console outputs. 

Reviewer: ```suggestion
As the editor doesn't support certain methods, you will be using the browser console to view console outputs. 
```

please pass your texts thorugh a grammar checker to make review easier later
